### PR TITLE
patrons: implement detail view

### PR DESCRIPTION
**Pull request by Garfield-fr**
Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Why are you opening this PR?

- Adds detail view.
- Adds link on brief view to patron name.
- Adds patron service.
- Adds translations.
- Fixes facet roles is open by default

## How to test?

- Check patron section on admin view

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+          {{ 'City' | translate }}:
+        </dt>
+        <dd class="col-sm-7 col-md-8 mb-0">
+          {{ record.metadata.postal_code }} {{ record.metadata.city }}
+        </dd>
+      </dl>
+      <!-- ROLES -->
+      <dl class="row mb-0">
+        <dt class="col-sm-3 offset-sm-2 offset-md-0">
+          {{ 'Role' | translate }}:
```
Comment (zannkukai) [inline projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html]: Typo : format plural if needed : 
```
<ng-container *ngIf="record.metadata.roles.length === 1; else roles" translate>Role</ng-container>
<ng-template #roles translate>Roles</ng-template>
```

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    }
+    return false;
+  }
+
+  get isLibrarian() {
+    if (this.record && this.record.metadata.roles) {
+      return this.record.metadata.roles.some((role: string) => role === 'librarian');
+    }
+    return false;
+  }
```
Comment (zannkukai) [inline projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.ts]: You already defined the function `isPatron()` into `projects/admin/src/app/record/brief-view/patrons-brief-view.component.ts` file. Is a a way to declare it once and reuse it (maybe part of a `patron.sevice.ts` file) ? (I'm not an Angular expert).

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html (hunk 1 above)
Comment (Garfield-fr) [reply projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html]: Plurial is in next line (singular for 1 role, plurial for many)

Comment (zannkukai) [reply projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html]: I propose a solution (found in `patrons-brief-view.component.ts`). Your code here, at this time, is :
```
<dt class="col-sm-3 offset-sm-2 offset-md-0">
  {{ 'Role' | translate }}:
</dt>
```
Should be 
```
<dt class="col-sm-3 offset-sm-2 offset-md-0">
  <ng-container *ngIf="record.metadata.roles.length === 1; else roles>{{ 'Role' | translate }}</ng-container>
  <ng-template #roles>{{ 'Roles' | translate }}</ng-template>
</dt>

Comment (Garfield-fr) [reply projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.html]: done

File: projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.ts (hunk 2 above)
Comment (Garfield-fr) [reply projects/admin/src/app/record/detail-view/patron-detail-view/patron-detail-view.component.ts]: done

**Pull request merged**